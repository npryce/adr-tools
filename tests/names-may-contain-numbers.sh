adr new 99 Luft Balloons
adr new Song 2
adr new 1999
adr new 9 to 5
ls doc/adr
