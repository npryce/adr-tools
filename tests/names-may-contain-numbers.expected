adr new 99 Luft Balloons
doc/adr/0001-99-luft-balloons.md
adr new Song 2
doc/adr/0002-song-2.md
adr new 1999
doc/adr/0003-1999.md
adr new 9 to 5
doc/adr/0004-9-to-5.md
ls doc/adr
0001-99-luft-balloons.md
0002-song-2.md
0003-1999.md
0004-9-to-5.md
